tests/ember-data-initializers.js

```diff
@@ -1,8 +1,9 @@
 /* eslint no-extra-semi: "off" */
 
+import DS from 'ember-data';
```

Comment:
Part of the reason why the build fails is this file is not transformed before it gets added to the bower build output. One thing we could do is remove this line and add  `var DS = require('ember-data').default;` inside the [iffe](https://en.wikipedia.org/wiki/Immediately-invoked_function_expression).